<?xml version="1.0"?>
<launch>

  <arg name="use_camera" default="true"/>
  <arg name="distance_threshold" default="3"/>
  <arg name="expiry_time" default="3"/>
  <arg name="camera_range" default="40"/>

  <group ns="wamv">


    <node name="perception_object_server" type ="object_server0.py" pkg="vrx_vision" output="screen">
      <remap from="map" to="perception_map"/>
      <param name="threshold" value="40"/>
      <param name="use_camera" value="$(arg use_camera)"/>
      <param name="expiry_time" value="$(arg expiry_time)"/>
      <param name="distance_threshold" value="$(arg distance_threshold)"/>
      <param name="camera_range" value="$(arg camera_range)"/>
      <remap from="objects" to="perception_objects"/>
    </node>

        
  </group>
</launch>
